sno/working_copy/gpkg.py

```diff
@@ -9,10 +9,11 @@
 import click
 import pygit2
 from osgeo import gdal
+from sqlalchemy import table, column
```

Comment:
`table`/`column` names seem very likely to be shadowed in local functions, leading to confusion.

A common pattern is to `import sqlalchemy as sa` and then use `sa.table` / `sa.column` everywhere, might be worth doing?

Reply:
Done